### PR TITLE
Analytics fix demo

### PR DESCRIPTION
**Pull request by charlietfe**

### CODE REVIEW

File: asker-analytics/backend/src/config/config.ts [hunk 1]
```diff
@@ -8,24 +8,26 @@ Booster.configure('on-aws', (config: BoosterConfig): void => {
...
+                  text: 'text',
+                },
+              },
+            ],
+          },
         ],
-        secretArn: 'arn:aws:secretsmanager:us-east-1:515849006004:secret:AskerSecretProducer-Ezrc1w'
+        producerConfig: [],
+        bootstrapServers: ['fast-caboose-01.srvs.cloudkafka.com:9094'],
+        secretArn: 'arn:aws:secretsmanager:us-east-1:515849006004:secret:AskerSecretProducer-Ezrc1w',
```
Comment (rdiaz82) [inline asker-analytics/backend/src/config/config.ts]: Just a reminder that this secret needs to be changed by the meeting presenter by his own secret

File: asker-analytics/frontend/src/index.tsx [hunk 2]
```diff
@@ -14,15 +14,14 @@ import { WebSocketLink } from '@apollo/client/link/ws';
...
+  uri: 'wss://da9fyn0wck.execute-api.us-east-1.amazonaws.com/on-aws/',
   options: {
     reconnect: true,
   },
 });
 
 const httpLink = new HttpLink({
-  uri:
-    'https://emc08t1htb.execute-api.eu-west-1.amazonaws.com/production/graphql',
+  uri: 'https://ediw2m695e.execute-api.us-east-1.amazonaws.com/on-aws/graphql',
```
Comment (rdiaz82) [inline asker-analytics/frontend/src/index.tsx]: This urls needs to be changed for the actual url

File: asker-analytics/frontend/src/index.tsx [hunk 3]
```diff
@@ -14,15 +14,14 @@ import { WebSocketLink } from '@apollo/client/link/ws';
 import { getMainDefinition } from '@apollo/client/utilities';
 
 const wsLink = new WebSocketLink({
-  uri: 'wss://ag3snnnxyi.execute-api.eu-west-1.amazonaws.com/production/',
+  uri: 'wss://da9fyn0wck.execute-api.us-east-1.amazonaws.com/on-aws/',
```
Comment (rdiaz82) [inline asker-analytics/frontend/src/index.tsx]: This urls needs to be changed for the actual url

File: asker/backend/src/config/config.ts [hunk 4]
```diff
@@ -16,14 +21,12 @@ Booster.configure('on-aws', (config: BoosterConfig): void => {
...
+              text: 'text',
+            },
           },
         ],
-        bootstrapServers: [
-          'fast-caboose-01.srvs.cloudkafka.com:9094',
-        ],
-        secretArn: 'arn:aws:secretsmanager:us-east-1:515849006004:secret:AskerSecretProducer-Ezrc1w'
+        bootstrapServers: ['fast-caboose-01.srvs.cloudkafka.com:9094'],
+        secretArn: 'arn:aws:secretsmanager:us-east-1:515849006004:secret:AskerSecretProducer-Ezrc1w',
```
Comment (rdiaz82) [inline asker/backend/src/config/config.ts]: This secret will need to be changed

File: asker-analytics/backend/src/config/config.ts (hunk 1 above)
Comment (charlietfe) [reply asker-analytics/backend/src/config/config.ts]: Let's use env variables for that

**Review by rdiaz82 — APPROVED** [review]
LGTM just some reminders about urls and secret that need to be changed before the meeting. could it owrth to move all those configurations into ENV. vars or an env file?

**Pull request merged**